nil/internal/collate/syncer.go

```diff
@@ -157,6 +159,7 @@ func (s *Syncer) Run(ctx context.Context, wgFetch *sync.WaitGroup) error {
 	s.logger.Debug().
 		Stringer(logging.FieldBlockHash, hash).
 		Uint64(logging.FieldBlockNumber, uint64(block.Id)).
+		Uint32(logging.FieldShardId, uint32(s.config.ShardId)).
```

Comment:
It already contains this field

https://github.com/NilFoundation/nil/pull/191/files#diff-ab230848b41511019d4484b9165904556e6b86d41eef2bf72c8279933b73b1b8L63